msal/src/main/java/com/microsoft/identity/client/NativeAuthPublicClientApplication.kt

```diff
@@ -86,6 +96,73 @@ class NativeAuthPublicClientApplication(
         //  To avoid duplicating the code, callback methods are routed through their
         //  coroutine-equivalent through this CoroutineScope.
         val pcaScope = CoroutineScope(SupervisorJob() + Dispatchers.IO)
+
+        fun getCurrentAccountInternal(config: NativeAuthPublicClientApplicationConfiguration): IAccount? {
+            LogSession.logMethodCall(TAG, "${TAG}.getCurrentAccountInternal")
+
+            val params = CommandParametersAdapter.createCommandParameters(
+                config,
+                config.oAuth2TokenCache
+            )
+
+            val command = GetCurrentAccountCommand(
```

Comment:
I think we did not add the GetCurrentAccountCommand class yet right? I think we should include it in the sign in round of PRs, since it's already being used in this PR. Or remove it's use from this PR and add it later.

Reply:
This class is pre-existing https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/common/src/main/java/com/microsoft/identity/common/internal/commands/GetCurrentAccountCommand.java